### ISSUE TITLE
Device beeps 3 times during boot when no display cable is connected (but boots fine)

### ISSUE THREAD

**Issue by silentcreek** [R3NE07/Futro-S740#34]
This is not an issue with this repository, but rather a question about an issue I'm experiencing with my system. Maybe someone else knows how to solve this and it might be worth adding the solution to this repository.

So, I have a Futro S740 with the latest Debian (12.1) installed. It works just fine. When I remove the display cable, however, the device beeps 3 times during boot. It contiues to boot just fine without any interaction needed, but the beeping is annoying during restarts, for example, so I'd like to know how to avoid them. If I have the display cable connected, the system boots silently without any beeps, but my goal is to run the system "headless".

I installed the latest UEFI firmware revision (1.13.0) and tried to change settings that I thought might be related, but that didn't help. Specifically, I disabled "Keyboard Error Reporting" during boot. I tried the "Quiet Boot" setting both enabled and disabled. And I changed the "Boot Error Handling" from "Pause and wait for key" to "Continue", but none of these solved my issue.

Any ideas?

**Comment by silentcreek** [comment]
So, I figured it out.

I tried various UEFI settings last night and it turns out the issue only occurs when "Secure Boot Control" is enabled. If Secure Boot is disabled, the system boots quietly regardless of a display cable plugged in. I don't understand why enabling Secure Boot would make the system complain when no display is connected, but at least now I know what causes the beeping.

So, if you want an entirely silent headless system, you either need to leave Secure Boot disabled (which is the default) or get a DisplayPort dummy plug to at least pretend a display is connected.

**Comment by silentcreek** [comment]
Just one more followup: In the meantime, I got a Fujitsu Futro S940 thin client. This shows the exact same behavior. But since this device comes with a serial port and console redirection, I hooked up a console cable to see if the UEFI firmware emits some error message during boot when it beeps (or maybe a hint how to disable the beeping). But that's not the case. The boot screen just looks like when you boot with a display attached. So, you just have to live with this behavior or disable Secure Boot or get a dummy plug.
